go/base/static/css/vumigo.css

```diff
@@ -14,8 +14,8 @@ button {
   pointer-events: auto;
 }
 .main-container {
+  width: auto;
   min-width: 650px;
-  max-width: none;
```

Comment:
Our max width rule was overriding a bootstrap rule that must've changed. Since it was no longer overriding anything, this rule became a no-op and we were left with a narrower width. `width: auto` brings back that overriding.